Chiazokam/Politico
# Ft datastructure create office 163356161

#### What does this PR do?
This PR tests and creates the Create Office Endpont

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint
- create a validator to test the office name input
- create a validator to test the office type input

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructure-create-office-163356161`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/offices`
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
	 "officeName": "Governor",
      "officeType": "state"
}
`
#### What are the relevant pivotal tracker stories?
[#163356161](https://www.pivotaltracker.com/story/show/163356161)

#### Images

![image](https://user-images.githubusercontent.com/26940294/51774429-9413a280-20f2-11e9-9d1c-e436e5ce48d4.png)
